docs/automatic-redirects.rst

```diff
@@ -33,7 +33,7 @@ Supported Top-Level Redirects
 .. note:: These "implicit" redirects are supported for legacy reasons.
           We will not be adding support for any more magic redirects.
           If you want additional redirects,
-          they should live at a prefix like :ref:`page-redirect`
+          they should live at a prefix like `Redirecting to a Page`_
```

Comment:
Why did you change these ones?

> Using ref is advised over standard reStructuredText links to sections (like `Section title`_) because it works across files, when section headings are changed, and for all builders that support cross-references.

from http://www.sphinx-doc.org/es/stable/markup/inline.html#role-ref

Reply:
mmm, well I don't like the idea of exposing a lot of links as global refs and more when they aren't used on the global scope, but I will revert those if you consider.

Reply:
I wonder if we can make a Sphinx extension that automatically ads each section heading as a global ref, prefixed by pagename or similar? That way you wouldn't need to declare all the refs, but still be able to reference them. That's definitely for another ticket/PR though.

Reply:
http://www.sphinx-doc.org/en/stable/markup/inline.html#role-numref looks interesting, via https://github.com/sphinx-doc/sphinx/pull/2963

Reply:
Hmm, looks like those are still for _explicit_ section refs, not implicitly creating them, ignore me :)

Reply:
Found it: http://www.sphinx-doc.org/en/stable/ext/autosectionlabel.html -- it's already built into Sphinx :)

Reply:
Like it!